### PR TITLE
Experiment with other ways of passing input to container.

### PR DESCRIPTION
**Pull request by mccalluc**
- It will continue to try to ssh until we turn that off.
- Docker containers won't be able to take advantage of the new mechanism until they are rewritten.

### CODE REVIEW

File: django_docker_engine/container_managers/docker_engine.py [hunk 1]
```diff
@@ -68,7 +68,11 @@ def _is_base_url_local(self):
             'http+docker://' + host for host in ['localunixsocket', 'localhost']
         ]
 
-    def run(self, image_name, cmd, **kwargs):
+    def run(self, image_name, cmd, environment={}, **kwargs):
+        assert 'environment' not in kwargs
```
Comment (scottx611x) [inline django_docker_engine/container_managers/docker_engine.py]: revert this

File: django_docker_engine/docker_utils.py [hunk 2]
```diff
@@ -35,9 +35,18 @@ class DockerClientWrapper():
     def __init__(self,
                  data_dir,
                  manager_class=docker_engine.DockerEngineManager,
-                 root_label='io.github.refinery-project.django_docker_engine'):
+                 root_label='io.github.refinery-project.django_docker_engine',
+                 # These do_input_json_* variables are redundant,
+                 # but until all the containers know to look at envvars,
+                 # this probably makes sense.
+                 do_input_json_file=True,
```
Comment (scottx611x) [inline django_docker_engine/docker_utils.py]: F F None

**Comment by mccalluc** [comment]
@scottx611x : If you want to re-review...

**Comment by mccalluc** [comment]
actually, I want to go through the user facing docs and make sure they're still accurate and the code snippets work.

**Pull request merged**